opm/common/OpmLog/LogBackend.hpp

```diff
@@ -75,6 +79,7 @@ namespace Opm
         int64_t m_mask;
         std::shared_ptr<MessageFormatterInterface> m_formatter;
         std::shared_ptr<MessageLimiter> m_limiter;
+        int64_t m_level;
```

Comment:
Is really `int64_t` the correct type for a level?  Why not `size_t` or `uint32_t`?

What about adding something like

    enum class LogLevel : size_t
    {
        Info = 0,
        Warning = 1,
        Error = 2
     };